Cisco/iOS-XE/snmp.conf

```diff
@@ -0,0 +1,9 @@
+! SNMP IOS-XE
+!
+! ACL for SNMP Access
+access-list 33 remark SNMP Query ACL
+access-list 33 permit 209.50.158.0 0.0.1.255
```

Comment:
Would add a line to permit polling from the EU platform:
`access-list 33 permit 193.177.128.0 0.0.3.255`
and users would choose to keep one or both

Reply:
Will do